content/guides/core/automations/automation-events.md

```diff
@@ -60,7 +60,7 @@ From the project's **Automations** tab or directly from a line plot panel, you c
 - A [system metric]({{< relref "/guides/models/app/settings-page/system-metrics.md" >}}) such as `cpu`, which tracks the percentage of CPU utilization. W&B logs system metrics automatically every 15 seconds.
 
 The notification can watch the metric for these events:
-- **Run metrics threshold met**: Trigger a workflow when for a given metric, a single logged value or the average logged values meets the threshold you specify.
+- **Run metrics threshold met**: Trigger a workflow when a single logged value for a metric, the average logged values for the metric over a number of runs, or the min or max value for the metric over a number of runs, meets the threshold you specify.
```

Comment:
I definitely think that introducing the phrase "over a number of runs" makes things more confusing. The average/min/max is calculated on the values from a single run. 

You use the phrase "logged value" for 2 of the 3, but just "value" for the 3rd, I assume that's a mistake. 

Would it be simpler to just say `Trigger a workflow when for a given metric, a single logged value or the average/min/max logged values meets the threshold you specify.`? Regardless, there's no reason to keep average and "min or max" in separate clauses of the sentence, it implies there's something different about them.

Reply:
PTAL, I restructured this area a bit more, to hopefully address this concern.